.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     name: Classifier cross-validation
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
```

Comment:
https://github.com/github-linguist/linguist/blob/master/.github%2Fworkflows%2Fci.yml

Reply:
> https://github.com/github-linguist/linguist/blob/master/.github%2Fworkflows%2Fci.yml

Please stop doing that.

Reply:
> https://github.com/github-linguist/linguist/# blob/- [ ] @[]()![WDR16BPBridgingtheDisabilityDividethroughDigitalTechnologyRAJA.pdf](https://github.com/user-attachments/files/15829183/WDR16BPBridgingtheDisabilityDividethroughDigitalTechnologyRAJA.pdf)

- - @ master/.github%2Fworkflows%2Fci.yml